python/cudf/cudf/core/column/column.py

```diff
@@ -290,8 +290,12 @@ def _concat(cls, objs, dtype=None):
 
         return col
 
-    def dropna(self):
-        dropped_col = self.as_frame().dropna()._as_column()
+    def dropna(self, drop_nan=True):
```

Comment:
Should we default to `drop_nan=False`? I don't think it's very common to want to drop both nulls and nans at the same time.

Reply:
Defaulted this to `False`